client/stylesheets/_toCat.scss

```diff
@@ -4832,16 +4834,26 @@ On click the images will be displayed at normal size to complete the effect
 	position:absolute;
 	left:0;
 	right:0;
+	border: 2px solid #f39d00;
```

Comment:
Color literals like `#f39d00` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs